src/gui/mzroll/forms/settingsform.ui

```diff
@@ -1443,6 +1449,7 @@ p, li { white-space: pre-wrap; }
  </tabstops>
  <resources>
   <include location="../mzroll.qrc"/>
+  <include location="../mzroll.qrc"/>
```

Comment:
@rish9511 extra include statement?

Reply:
@shubhra-agrawal 
Missed it somehow
Have pushed the necessary changes